src/mlpack/core/util/cli.hpp

```diff
@@ -2,7 +2,7 @@
  * @file cli.hpp
  * @author Matthew Amidon
  *
- * This file implements the CLI subsystem which is intended to replace FX.
+ * This file implements the CMD subsystem which is intended to replace FX.
  * This can be used more or less regardless of context.  In the future,
  * it might be expanded to include file I/O.
```

Comment:
Wow!  This comment was written in 2009 and hasn't changed.  Here's a more accurate suggestion if you don't mind updating it while we're at it:
```suggestion
 * This file implements the CMD subsystem which is a global singleton
 * intended to handle parameter passing for different binding types.
```